networks/testnet.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Name":"testnet",
+  "Purpose":"Evaluate Filecoin at a meaningful scale",
+  "Maintainer": "Protocol Labs",
+  "Stability": "high",
+  "Status": "offline",
+  "NetworkParameters":{
+    "SupportedProofTypes":[
+      {
+        "Name":"RegisteredSealProof_StackedDrg32GiBV1",
+        "Size":"34359738368",
+        "RegisteredSealProof": 3
+      },
+      {
+        "Name":"RegisteredSealProof_StackedDrg64GiBV1",
+        "Size":"68719476736",
+        "RegisteredSealProof": 4
+      }
+    ],
+    "ConsensusMinerMinPower": "10995116277760",
+    "EpochDurationSeconds": 30,
+    "ExpectedLeaderPerEpoch": 5,
+    "WPoStProvingPeriod": 2880,
+    "WPoStChallengeWindow": 60,
+    "WPoStPeriodDeadlines": 48,
+    "PreCommitChallengeDelay": 150
+  },
+  "ResourceLinks":[
+    {
+      "Name":"Faucet",
+      "Link":"https://faucet.testnet.filecoin.io/"
```

Comment:
@travisperson - The link https://faucet.glif.io works but this one doesn't redirect